var/spack/repos/builtin/packages/libpeas/package.py

```diff
@@ -75,7 +75,7 @@ def setup_build_environment(self, env):
         # LDFLAGS.
         pkg_config = which('pkg-config')
         python_prefix = self.spec['python'].prefix.lib.pkgconfig
-        python_pc_file = os.path.join(python_prefix, 'python3.pc')
+        python_pc_file = os.path.join(python_prefix, 'python3-embed.pc')
```

Comment:
Was this not working before? Or are we changing between two existing files?

Reply:
The latest version of python when libpeas was added was 3.7.4, so I think it worked.
I would like to fix it as follows:
$ diff package.py.old package.py
40c40
<     depends_on('python@3:', type='build')
---
>     depends_on('python@3:3.7.9', type='build')
78c78
<         python_pc_file = os.path.join(python_prefix, 'python3-embed.pc')
---
>         python_pc_file = os.path.join(python_prefix, 'python3.pc')
89c89
<         aclocal_pkg_list = ['pkgconf',
---
>         aclocal_pkg_list = ['pkgconfig',